crates/cheatcodes/src/config.rs

```diff
@@ -322,393 +251,6 @@ impl Default for CheatsConfig {
     }
 }
 
-// Helper function to set default chains
-fn create_default_chains() -> HashMap<String, ChainData> {
-    let mut chains = HashMap::new();
```

Comment:
This is entirely unused, I don't think https://github.com/foundry-rs/foundry/pull/10226 is wired up correctly